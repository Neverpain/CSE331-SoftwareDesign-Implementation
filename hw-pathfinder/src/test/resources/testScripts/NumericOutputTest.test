# Tests that the driver methods outputs all numerical values to 3 decimals
# digits and rounds properly

CreateGraph g

AddNode g A
AddNode g B
AddNode g C

AddEdge g A B 1
AddEdge g B C 3.3338

ListChildren g A

FindPath g A C

