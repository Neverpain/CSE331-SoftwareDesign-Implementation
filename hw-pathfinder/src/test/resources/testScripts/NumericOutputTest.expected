# Tests that the driver methods outputs all numerical values to 3 decimals
# digits and rounds properly

created graph g

added node A to g
added node B to g
added node C to g

added edge 1.000 from A to B in g
added edge 3.334 from B to C in g

the children of A in g are: B(1.000)

path from A to C:
A to B with weight 1.000
B to C with weight 3.334
total cost: 4.334